Maths/MidpointIntegration.js

```diff
@@ -42,7 +42,9 @@ function integralEvaluation (N, a, b, func) {
   // Calculate the integral
   let result = h
   temp = 0
-  for (let i = 0; i < pointsArray.length; i++) temp += pointsArray[i]
+  pointsArray.forEach(point => {
```

Comment:
Perhaps even better would be to use [`reduce`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) to find the sum.